test/e2e/e2ethanos/services.go

```diff
@@ -75,6 +80,7 @@ func NewPrometheus(sharedDir string, name string, config, promImage string) (*e2
 			"--storage.tsdb.max-block-duration": "2h",
 			"--log.level":                       infoLogLevel,
 			"--web.listen-address":              ":9090",
+			"--enable-feature":                  strings.Join(enableFeatures, ","),
```

Comment:
This flag should be added only if `len(enableFeatures) > 0`. Let's update it and see if E2E tests work or not